src/main.cpp

```diff
@@ -1094,8 +1094,8 @@ QString UsageArguments ( char** argv )
            "Client only:\n"
            "  -c, --connect         connect to given Server address on startup\n"
            "  -j, --nojackconnect   disable auto JACK connections\n"
-           "  -M, --mutestream      starts the application in muted state\n"
-           "      --mutemyown       mute me in my personal mix (headless only)\n"
+           "  -M, --mutestream      prevent others on a server from hearing what I play\n"
+           "      --mutemyown       prevent me from hearing what I play in the server mix\n"
```

Comment:
Does it still work headless only?

Reply:
Oh yes, I forgot about than... (mindless c-n-p...).

I have it on my long list of things to do to investigate how to make it work with the GUI.  It'll help the JSON-RPC interface, too.

Reply:
OK, updated and I also updated the "confirmation text" the program issues during start up.